source/Command.cpp

```diff
@@ -212,9 +212,21 @@ const string &Command::Description() const
 // a combination of more than one command, an empty string is returned.
 const string &Command::KeyName() const
 {
-	static const string empty;
+	static const string empty = "(none)";
 	auto it = keyName.find(*this);
-	return (it == keyName.end() ? empty : it->second);
+
+	return (!HasBinding() ? empty : it->second);
+}
+
+// Check if the key has no binding.
+bool Command::HasBinding() const
+{
+	auto it = keyName.find(*this);
+
+	if(it == keyName.end() || it->second == "")
+		return false;
+
+	return true;
```

Comment:
If we're going to combine the conditions into a single expression, just skip the if statements and return the result of the expression directly.
If we're using if statements for readability, then the conditions should be simpler expressions.

Pick one:

```suggestion
	if(it == keyName.end())
		return false;
	if(it->second.empty())
		return false;
	return true;
```

```suggestion
	return !(it == keyName.end() || it->second.empty());
```